documentation/components/Carousel.mdx

```diff
@@ -410,3 +410,14 @@ hideControls prop removes the control buttons from view until hovered.
     <div style={{ backgroundColor: ' rgb(0,204,0)', width: '700px' }}>Slide 3</div>
   </Carousel>
 </Playground>
+
+## Show Half Next and Prev Slides with infinite using set width
```

Comment:
At a glance I'm noticing an issue with the drag functionality for this example, but tomorrow I'll take a closer look at the updates made and see what could be the cause

Reply:
Yeah since I didn't modify the dragging but all of my changes involves set width slides, it might be that the draggable areas are much smaller now? like what happens when you try to drag in between the slides is more likely to happen, if you have a suggestion please let me know

Reply:
Ahh so I think the `handleDragActionHandler` function will need updating to account for the variable width slides while the carousel is also infinite. I think this will depend on the the slider measurements that are calculated when showHalfSlides AND infinite are true. 

When I added the variableWidth property, I did not account for infinite functionality. So what you see written here only accounts for non-variable width slides.. I'll continue looking into this and see if I can provide more suggestions. But wanted to share with you this initial find. 
![image](https://user-images.githubusercontent.com/11052295/116727555-54f7ea00-a9aa-11eb-9f96-19109214bece.png)